### ISSUE TITLE
Support for MongoDB v4

### ISSUE THREAD

**Issue by antony** [mixmaxhq/mongo-cursor-pagination#53]
Is this on your upgrade roadmap?

I actually started the process but there are a fair few failing tests so it might take some time to migrate. When trying to use the current codebase against v4, the pagination seems to not work at all - you can never request any page other than the first one (as it returns empty).

**Comment by Dj-jom2x** [comment]
yup same I just try now. same result even if i pass the same query with next string on it

**Comment by Dj-jom2x** [comment]
nvm it works xD
